src/components/organisms/AddEducationModal/index.tsx

```diff
@@ -0,0 +1,276 @@
+      checkAll();
+    };
+    return (
+      <DialogContainer
+        onClose={handleModal}
+        aria-labelledby="customized-dialog-title"
+        open={open}
+      >
+        <DialogTitle>
+          <Grid item>
+            <Title>
+              <Typography variant="body1" color={theme.palette.text.secondary}>
+                Add Certificate
+              </Typography>
+              <CloseButton onClick={handleModal}>
+                <CloseIcon />
+              </CloseButton>
+            </Title>
+          </Grid>
+        </DialogTitle>
+        <DialogContent>
+          <InputGrid container>
+            <FormContainer item>
+              <Typography
+                variant="overline2"
+                color={theme.palette.text.secondary}
+                sx={{ marginLeft: "10px" }}
+              >
+                Name of the University
+              </Typography>
+              <AutoCompleteDropdown
+                onChange={onUnviersityChange}
+                label="Search University Here"
+                data-testid="unviersityChange"
+                item_array={UNIVERSITY_ARRAY}
+              />
+            </FormContainer>
+            <FormContainer item>
+              <Box>
+                <Typography
+                  variant="overline2"
+                  color={theme.palette.text.secondary}
+                  sx={{ marginLeft: "10px" }}
+                >
+                  Degree
+                </Typography>
+              </Box>
+              <AutoCompleteDropdown
+                onChange={onDegreeChange}
+                label="Search Degree Here"
+                data-testid="degreeChange"
+                item_array={DEGREE_ARRAY}
+              />
+            </FormContainer>
+            <FormContainer item>
+              <Box>
+                <Typography
+                  variant="overline2"
+                  color={theme.palette.text.secondary}
+                  sx={{ marginLeft: "10px" }}
+                >
+                  Field of study
+                </Typography>
+              </Box>
+              <AutoCompleteDropdown
+                onChange={onFieldOfStudyChange}
+                label="Search Field Here"
+                data-testid="fieldChange"
+                item_array={FIELD_OF_STUDY}
+              />
+            </FormContainer>
+            <FormContainer item>
+              <Grid container columnGap="1.25rem">
+                <Grid item>
+                  <Box>
+                    <Typography
+                      variant="overline2"
+                      color={theme.palette.text.secondary}
+                      sx={{ marginLeft: "10px" }}
+                    >
+                      Start year
+                    </Typography>
+                  </Box>
+                  <DatePicker>
+                    <ResponsiveDatePickers
+                      setChanged={setStartDate}
+                      date={new Date()}
```

Comment:
if only month and year are required in MM/YYYY format then directly send the date in this format.  Or else create a variable for instance of data and reuse it, instead of recreating it again below